problemsets-mat223/modules/module15.tex

```diff
@@ -111,7 +118,27 @@
 \begin{example}
 	Find the eigenvectors/values of $A=\mat{1&2\\3&2}$.
 
-	XXX Finish
+	Similarly to the above example, we first determinate $\Char(A)$:
+	\begin{align*}
+	    \Char(A)&=\det\left(\mat{1-\lambda&2\\3&2-\lambda}\right)\\
+	            &=(1-\lambda)(2-\lambda)-6=\lambda^2-3\lambda-4=(\lambda-4)(\lambda+1)
```

Comment:
Let's factor the other way `(4-\lambda)(-1-\lambda)`. This way we avoid having an extra `-` sign when doing 3x3 matrices.